projects/robots/bitcraze/crazyflie/controllers/crazyflie/pid_controller.c

```diff
@@ -41,6 +41,7 @@ float constrain(float value, const float minVal, const float maxVal) {
 
 double pastAltitudeError, pastPitchError, pastRollError, pastYawRateError;
 double pastVxError, pastVyError;
+double altitudeIntergrator;
```

Comment:
There is still a typo here:
```suggestion
double altitudeIntegrator;
```